### PR TITLE
Read latest version from LATEST_RELEASE file

### PR DESCRIPTION
**Pull request by ibrahima**
Addresses #62

### CODE REVIEW

**Comment by ibrahima** [comment]
It seems like specs are failing because the fixture data for the Google Code pages is out of date. Not sure how best to update those. It also is weird that `open_uri(...)` returns a string instead of something that responds to read, it seems like the specs are kind of mocking the wrong API?

File: spec/google_code_parser_spec.rb [hunk 1]
```diff
@@ -21,7 +28,7 @@
 
   describe "#newest_download_version" do
     it "returns the last URL for the platform" do
-      expect(parser.newest_download_version).to eq Gem::Version.new("2.4")
+      expect(parser.newest_download_version).to eq Gem::Version.new("2.42")
     end
 
     context "out-of-order versions" do
```
Comment (ibrahima) [inline spec/google_code_parser_spec.rb]: This context is pretty unimportant now that it's just reading from a file, but I left it in for now. Let me know if I should just get rid of it!

**Comment by flavorjones** [comment]
Thanks! Looking at this now.

File: spec/google_code_parser_spec.rb (hunk 1 above)
Comment (flavorjones) [reply spec/google_code_parser_spec.rb]: Agree! I'll delete it.

**Comment by flavorjones** [comment]
Merged, will be in v2.0.2 which I'm cutting shortly.

**Comment by ibrahima** [comment]
Sweet, thanks!

**Comment by flavorjones** [comment]
Correction, the new release will be v2.1.0.

**Pull request merged**